lib/Elastica/Search.php

```diff
@@ -40,26 +40,19 @@ class Search
     public const OPTION_SEARCH_IGNORE_UNAVAILABLE = 'ignore_unavailable';
 
     /**
-     * @var BuilderInterface
+     * @var BuilderInterface|null
      */
-    private $_builder;
+    private $builder;
```

Comment:
What the strategy for this kind of renaming. Are we allowed to rename all private properties or as it's a new major version can we apply it to protected and public ones?

Reply:
@ruflin asked to have a new PR with the changes related to public/protected methods :)
Private ones should be fine, with the protected ones (and public too) I would change them before releasing the new major 7.0 release. wdyt?

Reply:
Yes it would be nice to be consistent.

I have a big PR on https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1569 for bringing support for elastica 7.x.
This Symfony bundle relies on some protected property like the `_name` of the Index class, so we have a room for changing that before publishing the GA release?

Reply:
Yes, timing before 7.0 GA seems to be good. Can we do this change potentially in an automated manner with the linter?

Reply:
IIRC the linter (php-cs-fixer) does not rename properties...